test/e2e/configurations/ci_e2e_config.yaml

```diff
@@ -6,11 +6,14 @@ pvcstress:
 ioSoakTest:
   replicas: 1
   duration: 10m
-  loadFactor: 2
+  loadFactor: 21
   protocols:
   - nvmf
 # - iscsi
-  fioFixedDuration: 60
```

Comment:
Maybe an indication of units would be helpful here too?

Reply:
Oops, that setting has been removed.

Reply:
That line has been deleted.